integration/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
     // Apply the application plugin to add support for building a CLI application in Java.
     id 'application'
     // Terra Test Runner Plugin
-    id 'bio.terra.test-runner-plugin' version '0.0.9-SNAPSHOT'
+    id 'bio.terra.test-runner-plugin' version '0.0.10-SNAPSHOT'
```

Comment:
I posed this question on slack, but did not get an answer. Where is this plugin published?

Reply:
It's published to [https://broadinstitute.jfrog.io/ui/repos/tree/General/libs-snapshot-local%2Fbio%2Fterra%2Fterra-test-runner%2F0.0.10-SNAPSHOT%2Fterra-test-runner-0.0.10-20211118.152429-70.pom](https://broadinstitute.jfrog.io/ui/repos/tree/General/libs-snapshot-local%2Fbio%2Fterra%2Fterra-test-runner%2F0.0.10-SNAPSHOT%2Fterra-test-runner-0.0.10-20211118.152429-70.pom).